captum/optim/_core/objectives.py

```diff
@@ -147,23 +150,31 @@ def optimize(
         return history
 
 
-def n_steps(n: int) -> StopCriteria:
+def n_steps(n: int, show_progress: bool = True) -> StopCriteria:
     """StopCriteria generator that uses number of steps as a stop criteria.
     Args:
         n (int):  Number of steps to run optimization.
+        show_progress (bool):  Whether or not to show progress bar.

```

Comment:
nit: show_progress (bool, optional)
      Default: True

Reply:
Okay, I've made the change!